var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -266,8 +300,75 @@ def configure_args(self):
         if sys.platform == 'darwin':
             options.append('--with-build-config=bootstrap-debug')
 
+        # nvptx-none offloading for host compiler
+        if spec.satisfies('+nvptx'):
+            options.extend(['--enable-offload-targets=nvptx-none',
+                            '--with-cuda-driver-include={0}'.format(
+                                spec['cuda'].prefix.include),
+                            '--with-cuda-driver-lib={0}'.format(
+                                spec['cuda'].libs.directories[0]),
+                            '--disable-bootstrap',
+                            '--disable-multilib'])
+
         return options
 
+    # run configure/make/make(install) for the nvptx-none target
+    # before running the host compiler phases
+    @run_before('configure')
```

Comment:
maybe this should only be done for `+nvptx`?

Reply:
I think it is not possible to say `@run_before(..., when="+nvptx")`, or do you have something else in mind?

Reply:
I think
```python
@when('+nvptx')
@run_before('configure')
```
might work :)

Reply:
Makes sense :-)

Reply:
yeah, i think that will work. will try to get that tested & updated later today or this weekend

Reply:
Fixed in commit 5252c10.